vulcanai2/models/layers.py

```diff
@@ -52,8 +53,8 @@ def _init_weights(self):
         if self.initializer is None, then pytorch default weight
```

Comment:
@joemehltretter ^ also please update the example script since this changes the config dict required to build the layer ```dense_config = {
    'dense_units': [100, 50],
    'initializer': None,
    'bias_init': None,
    'norm': None,
    'dropout': 0.5,  # Single value or List
}```
should turn into:
dense_config = {
    'dense_units': [100, 50],
    'weight_init': None,
    'bias_init': None,
    'norm': None,
    'dropout': 0.5,  # Single value or List
}